{
  "path": "/documentation/cfnetwork/cfnetworkerrors/cferrorhttpbadproxycredentials",
  "type": "Enumeration Case",
  "name": "CFNetworkErrors.cfErrorHTTPBadProxyCredentials",
  "desc": "The proxy rejected the authentication credentials provided for logging in.",
  "items": [],
  "declaration": "case cfErrorHTTPBadProxyCredentials = 307"
}